plugins/tabular_dataset_chunk.h

```diff
@@ -102,6 +102,16 @@ struct TabularDatasetChunk {
         }
     }
 
+    RowName getRowName(size_t index) const
+    {
+        return rowNames.at(index);
+    }
+
+    const RowName & getRowName(size_t index, RowName & storage) const
```

Comment:
It's for compatibility with the next PR, which has that interface (it's a way to return something that can be a reference if it already exists, or will outlive the function if not, without doing an allocation).